test/metabase/actions/test_util.clj

```diff
@@ -35,7 +35,7 @@
         (reset! db (mt/db))
         (thunk))
       (finally
-        (let [{driver :engine, db-id :id} @db]
+        (when-let [{driver :engine, db-id :id} @db]
```

Comment:
Minor fix -- if a test DB can't be created because it runs into an error, don't try to destroy the DB in the `finally` block. I ran into this when the `postgres` service on my computer shut down after running `apt upgrade`